docs/src/main/sphinx/connector/bigquery.rst

```diff
@@ -45,24 +45,22 @@ that should generally lead to better read performance:
 Requirements
 ------------
 
-Enable the BigQuery Storage API
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Follow `these instructions <https://cloud.google.com/bigquery/docs/reference/storage/#enabling_the_api>`_.
-
-Authentication
-^^^^^^^^^^^^^^
-
-**On GCE/Dataproc** the authentication is taken from the machine's role.
-
-**Outside GCE/Dataproc** you have 3 options:
-
-* Use a service account JSON key and ``GOOGLE_APPLICATION_CREDENTIALS`` as
-  described `here <https://cloud.google.com/docs/authentication/getting-started>`_.
-* Set ``bigquery.credentials-key`` in the catalog properties file.
-  It should contain the contents of the JSON file, encoded using base64.
-* Set ``bigquery.credentials-file`` in the catalog properties file.
-  It should point to the location of the JSON file.
+* Enable the `BigQuery Storage Read API
+  <https://cloud.google.com/bigquery/docs/reference/storage/#enabling_the_api>`_.
+* Set up authentication. Your authentiation options differ depending on whether
+  you're using Dataproc/Google Compute Engine (GCE) or not.
+
+  **On Dataproc/GCE** the authentication is taken from the machine's role.
+
+  **Outside Dataproc/GCE** you have 3 options:
+
+  * Use a service account JSON key and ``GOOGLE_APPLICATION_CREDENTIALS`` as
+    described in the Google authentication `getting started guide
```

Comment:
Maybe?
```suggestion
    described in the Google Cloud `getting started guide
```

Reply:
Problem is it's specifically a getting started guide to authentication, not Cloud.

How about "..in the Google Cloud authentication `getting started guide..."?

Reply:
"..in the Google Cloud authentication `getting started guide..." is much better. Thanks